sample/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ android {
 
   defaultConfig {
     applicationId = namespace
-    minSdk = 31
+    minSdk = libs.versions.minSdk.get().toInt()
```

Comment:
do you really need to run the sample app on all API levels? is having the library support 21 not sufficient?

Reply:
I tested the sample app on my real device, which is 29 API version, and I thought it would be better to downgrade the version for other people who might test it on lower API versions

Reply:
that's fair! let's keep this then.